src/pages/resultsView/querySummary/PatientSampleSummary.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { SampleIdentifier, Patient, Sample } from 'cbioportal-ts-api-client';
+import _ from 'lodash';
+
+interface IPatientSampleSummaryProps {
+    samples: Sample[] | SampleIdentifier[];
```

Comment:
@gblaih i went ahead and merged but probably we should just pass in counts instead of this slightly odd signature.  But it's not important enough to bother.